source/change-streams/tests/change-streams.yml

```diff
@@ -32,7 +32,17 @@ tests:
           command_name: aggregate
           database_name: *database_name
     result:
-      success: []
+      success:
+        -
+          _id: "42"
+          documentKey: "42"
+          operationType: insert
+          ns:
+            db: *database_name
+            coll: *collection_name
+          fullDocument:
+            x:
+              $numberInt: "1"
```

Comment:
I think this should be `x:1` instead of `$numberInt: "1"` but it looks like all the other tests already use `$numberInt` so lgtm.

Reply:
I'm inclined to agree, but didn't want to risk anything.